src/pal/src/locale/utf8.cpp

```diff
@@ -1168,24 +1168,23 @@ class UTF8Encoding
         return count;
     }
 
-    int GetBytesUnknown(BYTE* pSrc, int ch, BYTE **bytesUnknown)
+    int GetBytesUnknown(BYTE* pSrc, int ch, BYTE *bytesUnknown)
```

Comment:
```suggestion
    int GetBytesUnknown(BYTE* pSrc, int ch, BYTE* bytesUnknown)
```
to be consistent with other places.